src/styles.py

```diff
@@ -165,6 +180,28 @@ def _on_style_change(self, override=False):
             "sensitive", canvas, "highlight_enabled", 2,
         )
 
+    def _generate_preview(self, style: dict) -> Gio.File:
+        with rc_context(style):
+            # set render size in inch
+            figure = Figure(figsize=(6, 3), dpi=300)
```

Comment:
This requires a bit of fine tuning when we implement the previews in UI

Reply:
Right, I see now that you actually made the same comment as I did previously about the figure size maybe needing some finetuning based on the UI.